apps/web/src/app/host-dashboard/page.tsx

```
@@ -0,0 +1,1131 @@
+              >
+                <X className="w-6 h-6" />
+              </button>
+            </div>
+
+            <form onSubmit={handleAddProperty} className="space-y-6 ">
+              {/* Basic Information */}
+              <div className="border-b border-gray-200 pb-6">
+                <h3 className="text-lg font-semibold dark:text-white text-gray-900 mb-4">Basic Information</h3>
+                <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+                  <div className="md:col-span-2">
+                    <label className="block text-sm font-medium dark:text-white text-gray-700 mb-2">Property Title *</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)